git/sync/sync-with-azure.yml

```
@@ -0,0 +1,21 @@
+
+jobs:
+  alert:
+    runs-on: ubuntu-latest
+    name: Sync workflow
+    steps:
+    - uses: actions/checkout@v3
+    - uses: a11smiles/GitSync@main
+      env:
+        ado_token: '${{ secrets.ADO_PERSONAL_ACCESS_TOKEN }}'
+        github_token: '${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}'
+        config_file: './.github/workflows/sync_config.json'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)